core/platform/auth/firebase_auth_services.py

```diff
@@ -78,6 +79,14 @@
 transaction_services = models.Registry.import_transaction_services()
 
 
+class FireBaseClaims(typing_extensions.TypedDict):
+    """Dict type for Firebase Claim Dict."""
+
+    sub: str
```

Comment:
```suggestion
class AuthClaimsDict(typing_extensions.TypedDict):
    """Dictionary representing the AuthClaims object."""
    sub: str
```

and also move this class to _auth_domain.py_

Reply:
or you can move this to https://github.com/oppia/oppia/blob/develop/stubs/firebase_admin/auth.pyi#L113 and then return the `AuthClaimsDict` from `verify_session_cookie`.

Reply:
Done.

Reply:
How can i import AuthClaimDict from firebaseadmin
I tried 
```
import firebase_admin
firebase_admin.AuthClaimDict
```
but backend test fails

Reply:
You need to do the import in the `if MYPY:` block.

Reply:
That didn't work.
So what I did was instead of moving it to https://github.com/oppia/oppia/blob/develop/stubs/firebase_admin/auth.pyi
I defined AuthClaimDict in the init file and kept a copy of authclaimdict in this file due to import error. I tried all methods but didn't work.
Btw I will work on this today and let you know the updates

Reply:
I will move AuthClaimDict to auth_domain. That one works.

Reply:
Which backend tests fail, what is the error? Don't move stuff around until you're 100 % sure that it is not possible to fix the error.

Reply:
Done

Reply:
> Which backend tests fail, what is the error? Don't move stuff around until you're 100 % sure that it is not possible to fix the error.

Actually, I didn't move anything other than `authclaimdict` which is made by me. And backend test failed since I couldn't import firebase_admin, so since you mentioned that I can move `authclaimdict`(earlier `firebasedict`) to auth_domain I did so.